oelauncher.cmd

```diff
@@ -24,24 +24,21 @@ REM This can be extended by adding more ELSE IF statements for p1
 
 
 IF "%p1%" equ "forum" (
-    "C:\Program Files\CZM\FORUM Viewer\api\launchFORUM.cmd" -patientId %p2%
+    "C:\Program Files\FORUM Viewer\api\launchFORUM.cmd" -patientId %p2%
     goto exit
 ) ELSE IF "%p1%" equ "forumsop" (
-    "C:\Program Files\CZM\FORUM Viewer\api\launchFORUM.cmd" -sopInstanceUid %p2%
+    "C:\Program Files\FORUM Viewer\api\launchFORUM.cmd" -sopInstanceUid %p2%
     goto exit
 ) ELSE IF "%p1%" equ "forumdate" (
-    "C:\Program Files\CZM\FORUM Viewer\api\launchFORUM.cmd" -patientId %p2% -examDate %p3%
+    "C:\Program Files\FORUM Viewer\api\launchFORUM.cmd" -patientId %p2% -examDate %p3%
     goto exit
 ) ELSE IF "%p1%" equ "complog" (
     cd "C:\Program Files (x86)\COMPlog\COMPlog\"
     start "" "Complog.exe"
     goto exit
 )
 ELSE IF "%p1%" equ "imageNet" (
-    cd "C:\Topcon\"
-
-    (echo @VIEW& echo %p2%) > View.cmd
-    start "" "imagenet6view.bat"
+    "C:\Program Files\Google\Chrome\Application\chrome.exe" http://rhmtopcon/IMAGEnet/Patient/%p2%

```

Comment:
I assume rhmtopcon is a local alias, not an official URL or protocol?
In which case it cannot be hard coded. It will need to be parametized.

Also, there should be no need to explicitly call chrome.exe at a fixed location. you should just be able to use `start <urll>` which should open the URL in the default browser.
Or at the very least, just call `chrome.exe` - which would require chrome to be in the `PATH`